conjure-java-core/build.gradle

```diff
@@ -39,6 +39,7 @@ dependencies {
     compile 'com.google.guava:guava'
     compile 'com.palantir.conjure.java.api:errors'
     compile 'com.palantir.ri:resource-identifier'
+    compile 'com.palantir.safe-logging:preconditions'
```

Comment:
I think we'd need to add this to the `:conjure-lib` project to make sure the class is available for all users.